src/common/ProfileMenu.tsx

```diff
@@ -0,0 +1,125 @@
+import { config } from '@config'
+import {
+  Divider,
+  ListItemIcon,
+  Menu,
+  MenuItem,
+  Stack,
+  Tooltip,
+  Typography,
+  useTheme,
+} from '@mui/material'
+import { useMemo, useState } from 'react'
+import { redirect } from 'react-router-dom'
+
+import { UserAvatar } from '@/common'
+import { BusEvents, RoutePaths } from '@/enums'
+import { bus } from '@/helpers'
+import { useAuth, useMetamaskZkpSnapContext } from '@/hooks'
+import { UiIcon } from '@/ui'
+
+interface ProfileMenuProps {
+  anchorEl?: null | HTMLElement | undefined
+  handleClose: () => void
+}
+
+const MAX_LENGTH_DID = 12
```

Comment:
what does it mean? DID cannot be lnger than 12 characters?

Reply:
renamed, Displayed DID length is 12 characters or less